docs/guides/python-collector.md

```diff
@@ -0,0 +1,486 @@
+F<!--
```

Comment:
```suggestion
<!--
```

Reply:
[Press F ...](https://dotesports.com/culture/news/the-origin-of-press-f-to-pay-respects)